style/signup.css

```
@@ -0,0 +1,125 @@
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

❗️ 수정요청
고정값으로 높이를 가지게 되면서 화면 크기가 줄었을때 로고가 짤리네요. 가능하면 min-height로 변경하시는 것을 추천드려요!
```suggestion
    min-height: 100vh;
```